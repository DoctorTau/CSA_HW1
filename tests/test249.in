453
-1711 3967 9318 408 -4274 772 8859 3975 -9378 9264 4210 -3986 3204 725 1872 9874 -2964 6588 7915 6052 9017 7313 -3049 -2326 9255 -7824 -5582 864 -1813 1428 5290 -8780 1472 8655 -7567 -5598 585 206 -2146 9701 6547 -8000 -5943 1188 -9128 -9233 8501 8292 -9908 -6293 4290 -7641 -4567 5096 4982 9058 -9528 6229 9603 7118 8538 -1298 -965 1148 3659 -154 3884 -8357 -7701 5034 1327 -1551 986 -1990 -7536 -5174 -8393 -6198 -255 -9201 2788 -4378 -6192 8663 -9731 2721 5484 -7481 -6757 6468 -1136 9535 5594 -9171 -5920 -5005 -4657 7556 144 -7998 9026 -3112 5310 956 -6686 1335 5922 8978 7973 4240 8258 -5245 -9228 -2775 941 1719 -848 5537 5978 3786 9641 -2768 -1811 2206 7606 930 1136 -4804 -5052 2466 -453 5037 -993 -435 -6474 -7529 9466 9798 2486 3379 -519 1205 -2125 -6365 -570 -9782 -4176 645 -958 6281 -1665 1699 -71 -7893 3564 8024 8593 -8402 -3632 5035 4080 6866 -794 3463 6381 -7385 3955 6204 -2414 9678 3044 8746 -6229 -5686 5523 6879 3967 -7520 -1167 -1683 3352 970 3442 8598 -4363 4844 -3182 -3904 -7917 -7358 4802 8640 1885 -1790 -2341 7167 5450 1302 -3802 7976 5229 2504 -5933 9544 -1462 8463 -448 5486 -4426 1403 5995 -8820 8353 -3037 7560 2451 -3843 -4124 -7496 -948 -9269 794 -6728 -3183 4136 220 1043 7312 7673 4781 -4390 9416 -5227 -8387 -1310 -4909 -7065 8840 7618 -5774 1875 -7022 1759 1664 7612 -5977 -2636 2724 6852 -1412 -3470 -4839 -2422 584 9492 -9922 -2469 3951 8360 -8513 6872 -7876 -8535 3594 5103 -5455 2913 -40 -5198 -8970 -5719 9807 9647 -9189 -7669 1058 -9363 -9146 2775 4702 9479 1032 8238 4617 1268 -5874 1075 1588 -2605 8808 -6000 7318 -3131 5660 247 4924 -2921 1671 3127 -8129 2379 650 4167 -8971 5703 1336 8102 2612 -2830 3697 -9441 -380 1311 7028 -4198 5317 -1730 4418 -6393 -7269 -998 -4591 -543 -2248 1450 6165 9889 -5369 -1654 -1317 1939 685 -2518 -2643 -2115 5153 8620 1465 -5753 -9069 3690 -8730 -358 7524 8601 4231 -6322 -8643 -3133 -4691 -9130 -7997 1593 -4218 1984 2675 2753 -291 2474 -5091 -6237 -3511 -2024 -4217 3479 -2015 9386 -9184 6838 -4606 -3988 -1542 -1780 -4062 -9882 -3479 8660 2062 -7611 8681 -5050 4075 -1251 -9660 -3313 4277 3269 3907 -94 2331 -8692 4573 -9723 7512 -3848 6772 8820 -1752 7848 -9971 -9730 9161 -9437 2944 -9567 9418 -348 -1107 6041 -6179 1280 -1951 7206 948 -9237 -2372 1400 5422 6231 -6856 555 5736 2200 -9231 -9752 5757 -9377 -749 4400 7524 7471 -2569 7752 -2798 8422 -7684 8662 3567 -4065 5440 -5159 -2479 8280 5258 -7809 9601 374 8123 -7458 -7824 -8038 6377 -365 