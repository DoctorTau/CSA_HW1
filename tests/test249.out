-4274 -9378 -3986 -2964 -2326 -7824 -5582 -8780 -5598 -2146 -8000 -9128 -9908 -9528 -1298 -154 -1990 -7536 -5174 -6198 -4378 -6192 -1136 -5920 -7998 -3112 -6686 -9228 -848 -2768 -4804 -5052 -6474 -570 -9782 -4176 -958 -8402 -3632 -794 -2414 -5686 -7520 -3182 -3904 -7358 -1790 -3802 -1462 -448 -4426 -8820 -4124 -7496 -948 -6728 -4390 -1310 -5774 -7022 -2636 -1412 -3470 -2422 -9922 -7876 -40 -5198 -8970 -9146 -5874 -6000 -2830 -380 -4198 -1730 -998 -2248 -1654 -2518 -8730 -358 -6322 -9130 -4218 -2024 -9184 -4606 -3988 -1542 -1780 -4062 -9882 -5050 -9660 -94 -8692 -3848 -1752 -9730 -348 -2372 -6856 -9752 -2798 -7684 -7458 -7824 -8038 